### PR TITLE
Solved issue 116

### PR DESCRIPTION
**Pull request by ruchir594**
Hello,

I hope I solved the issue 116 the right way. Happy to receive feedback.

### CODE REVIEW

**Comment by seemethere** [comment]
#116 <-- So that the issue gets linked to this PR

File: sanic/request.py [hunk 1]
```diff
@@ -61,7 +62,10 @@ def json(self):
             try:
                 self.parsed_json = json_loads(self.body)
             except Exception:
-                pass
+                return HTTPResponse(json_dumps(self.body),
```
Comment (seemethere) [inline sanic/request.py]: I don't think it makes sense to return an HTTPResponse from a function in the request. What do you think @channelcat?

Comment (pahaz) [inline sanic/request.py]: there is a really bad idea. I don't expect to get the response in case to access the `request.json`

Comment (EFF) [inline sanic/request.py]: @pahaz agreed, makes no sens at all.

**Comment by seemethere** [comment]
Closing for now, we may open up back later.

**Pull request closed without merging**